README.md

```diff
@@ -1,82 +1,28 @@
-# GrimoireLab Tutorial
+# GrimoireLab User Guide
 
-[GrimoireLab](http://grimoirelab.github.io/) is free, open source software for software development analytics. It allows you to retrieve data from many kinds of systems with information related to software development, and produce analysis and visualizations with it.
+[GrimoireLab toolset](https://chaoss.github.io/grimoirelab) User Guide (GUG) for newcomers
 
-![](assets/wizard-y.svg)
+This documentation is using [Just the Docs](https://pmarsceill.github.io/just-the-docs/) template. You can use its documentation as reference.
 
-GrimoireLab supports more than 20 different kinds of [data sources](https://gitlab.com/Bitergia/c/FAQ/tree/master/sources#supported-data-sources), from git repositories or GitHub projects, to issue trackers such as Jira or Bugzilla, including messaging systems such as IRC, Slack or mailing lists, or other types of systems such as StackOverflow or Jenkins. 
+# How to set it up locally
 
-With the tools in the GrimoireLab toolset you can retrieve data, store it in databases, analyze it, and produce nice dashboards for interactively visualizing the resulting information.
+Clone this repository
 
-![](eclipse.png)
-
-Example dashboard produced with GrimoireLab: the [CHAOSS Development Dashboard](https://chaoss.biterg.io)
-
```

Comment:
I would mark this comment as marking.

I would revert the changes above this marking and move the below content to the `index.md` file. The below content i.e, quick overview of the grimoirelab tools would look good on the first page of the tutorial.